synapse/api/auth.py

```diff
@@ -326,8 +294,7 @@ async def _get_appservice_user_id_and_device_id(
              Must use `org.matrix.msc3202.device_id` in place of `device_id` for now.
 
         Returns:
-            3-tuple of
-            (user ID?, device ID?, application service?)
+            the application service ``Requester`` of that request
 
         Postconditions:
```

Comment:
this text about postconditions could do with rewording to be about the resultant `Requester`

Reply:
Done